tools/run_tests/artifacts/artifact_targets.py

```diff
@@ -161,8 +155,8 @@ def build_jobspec(self):
                 'tools/run_tests/artifacts/build_artifact_python.sh',
                 environ=environ,
                 timeout_seconds=60 * 60,
-                docker_base_image='quay.io/pypa/manylinux1_i686'
-                if self.arch == 'x86' else 'quay.io/pypa/manylinux1_x86_64')
+                docker_base_image='quay.io/pypa/{}_{}'.format(
```

Comment:
This seems a little fragile. What if we add support for another architecture? Can we pull this out to a helper function?

Reply:
Looks like the Fusion job failed. The only manylinux2010 image I see on quay.io is pypa/manylinux2010_x86_64.

Reply:
I rather decide to remove this because it's not necessary for builders to keep it updated. This should be done manually by engineers.